### DISCUSSION TITLE
I have 460 GB downloaded of a 1.2TB file from a  B2 Back blaze backup

### DISCUSSION THREAD

**Discussion by SimonFearby** [iterate-ch/cyberduck#15080] [Q&A]
Cyberduck 8.6.1 has downloaded 460GB ok of a 1.2 TB backblaze file in a B2 bucket file, I let my machine o ()no sleeping or power saving) and Chber Duck stopped the download while I was away, then I pressed to resume the download started again from byte 1

I stopped Cyberduck and instantly backed up the 460GB "simon_u_drive.zip.cyberducksegment"

[segment files.txt](https://github.com/iterate-ch/cyberduck/files/12503182/segment.files.txt)

How can I resume this without starting again?

My System is Windows 11 (latest nonbeta), 5950X, 62GB DDR4, PCIE4 ssd), no power savings of sleeping on the disk, monitor, pc on inactivity 

Cyberduck Log 
[cyberduck.log](https://github.com/iterate-ch/cyberduck/files/12503180/cyberduck.log)

Does cyberduck allow resuming a segment connection or rebuilding a transfer from C:\Users\%user%\AppData\Roaming\Cyberduck\Transfers

**Comment by tatze96** [comment]
The download should be resumed based on the completed segments if you download the file to the exact location and the previous segments are still there and intact. Cyberduck will typically only restart the download completely if the present segments are either incomplete or defective.
